app/controllers/general.php

```diff
@@ -1118,6 +1118,7 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
                 ->setParam('trace', $output['trace'] ?? []);
 
             $response->html($layout->render());
+            return;
```

Comment:
Improves quality, prevents further code doing duplicate response.
Doesnt affect logic now, thanks to HTTP filtering it, but still, quality matters.